pages/content/amp-dev/documentation/guides-and-tutorials/start/bento_guide.md

```diff
@@ -0,0 +1,102 @@
+---
+$title: Use AMP components in non-AMP pages
+$order: 1
+description: 'Use a subset of AMP components as standalone elements in non-AMP pages with Bento AMP.'
+tutorial: false
+toc: true
+formats:
+  - websites
+author: CrystalOnScript
+---
+
+Use a subset of AMP components as standalone elements in non-AMP pages with Bento AMP. Bento AMP is a project that embraces use of AMP components without the need to commit to fully valid AMP. This creates copy and past-able code for common UI widgets, allows testing if AMP is the right switch for your website, and using Bento in a migration plan. 
```

Comment:
Are we framing bento as a stepping stone to AMP?

Reply:
Last I heard it was a possible path - cc @caroqliu @dvoytenko

Reply:
I think we should frame bento bigger. My (badly phrased) suggestion for this paragraph would be:

Use AMP components everywhere with Bento AMP. Bento AMP allows you to use AMP components in non-AMP pages without the need to commit to fully valid AMP. For example. you can use AMP's well-tested and accessible carousel implementation with frameworks or CMSes that don't support AMP.

Reply:
Bento _can be_ a stepping stone to AMP, but it should be valuable to anyone regardless of their investment level in AMP. @nainar likely has good copy for how to frame this.

Reply:
Agree that a stepping stone is one of possible use cases. It's perfectly fine for people to stop at Bento AMP. 

I like Sebastian's phrasing, also from the blog:

`Developers can use Bento components for one-off cases such as adding a carousel to an otherwise non AMP page, or as a way of testing out AMP components on the path to being valid AMP. `

Reply:
updated the intro, would love to hear what ya'll think!